muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/ceepos/rest/CeeposRESTService.java

```diff
@@ -752,6 +781,59 @@ public Response removeeOrder(@PathParam("ORDERID") Long orderId) {
     return Response.noContent().build();
   }
   
+  @Path("/order/{ORDERID}/returnLink")
+  @GET
+  @RESTPermit(handling = Handling.INLINE)
+  public Response getReturnLink(@PathParam("ORDERID") Long orderId) {
```

Comment:
Should this have a check to make sure the logged user is the owner of the order or something?

Reply:
Added a safety check for order ownership.